src/ripple/rpc/impl/RPCHelpers.cpp

```diff
@@ -241,10 +241,13 @@ ledgerFromRequest(T& ledger, JsonContext& context)
         }
         else
         {
+            int iVal;
```

Comment:
I don't think it matters in the short term, but this should be a `std::uint32_t` to match the argument for `getLedger` and also catch "mistakes" like using negative ledger indices.

Also, I guess, to ensure this doesn't break sometime around 2235 when we hit 2 billion ledgers, because we will, invariably, be on vacation and it sucks to have your vacation interrupted because of an integer over/underflow.